webapp/lib/model.g.dart

```diff
@@ -60,6 +76,22 @@ class Conversation {
     return pubSubClient.publishDocChange(collectionName, docIds, {"tags": newValue?.toList()});
   }
 
+  Future<bool> removeFromTagIds(DocPubSubUpdate pubSubClient, String newValue) {
```

Comment:
I'm not sure about 'newValue' as the argument name - is it the name of the argument to remove?

Reply:
Good catch. Renamed to `valueToRemove` here and elsewhere.